### PR TITLE
Publicize: Add empty Composer package

### PR DESCRIPTION
**Pull request by danielpost**
This PR adds an empty shell package for Publicize. When #22511 is merged the updated files will go into this package.
#### Changes proposed in this Pull Request:

* Add an empty Composer package for Publicize.

#### Jetpack product discussion

p9dueE-4cw-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* N/A

### CODE REVIEW

**Review by pablinos — APPROVED** [review]
Looks good to me!

File: projects/packages/publicize/composer.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"branch-alias": {
```
Comment (jeherve) [inline projects/packages/publicize/composer.json:41]:
```suggestion
		"autotagger": true,
		"mirror-repo": "Automattic/jetpack-publicize",
		"textdomain": "jetpack-publicize-pkg",
		"changelogger": {
			"link-template": "https://github.com/Automattic/jetpack-publicize/compare/v${old}...v${new}"
		},
		"branch-alias": {
```

File: projects/packages/publicize/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# Files to include in the mirror repo, but excluded via gitignore
+# Remember to end all directories with `/**` to properly tag every file.
+# /src/js/example.min.js  production-include
+
+# Files to exclude from the mirror repo, but included in the monorepo.
+# Remember to end all directories with `/**` to properly tag every file.
+.gitignore        production-exclude
+changelog/**      production-exclude
+phpunit.xml.dist  production-exclude
```
Comment (jeherve) [inline projects/packages/publicize/.gitattributes:14]:
```suggestion
phpunit.xml.dist  production-exclude
.phpcs.dir.xml     production-exclude
```

**Review by jeherve — CHANGES_REQUESTED** [review]
This looks good. 

I would suggest adding a `.phpcs.dir.xml` file like the one in `projects/packages/search/.phpcs.dir.xml`, so we can manage i18n once we start adding to that package.

File: projects/packages/publicize/.gitattributes (hunk 2 above)
Comment (pablinos) [reply projects/packages/publicize/.gitattributes:14]: Handled in ccddf8b

File: projects/packages/publicize/composer.json (hunk 1 above)
Comment (pablinos) [reply projects/packages/publicize/composer.json:47]: Handled in ccddf8b

**Comment by pablinos** [comment]
Thanks @jeherve! I've added that in ccddf8b. I'll try and sort out the failing checks now, but if you have time to take another look, that would be great.

**Review by jeherve — APPROVED** [review]

**Pull request merged**